addons/website_sale_stock/tests/test_website_sale_stock_multilang.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import tagged
+from odoo.tests.common import HttpCase
+
+
+@tagged('post_install', '-at_install')
+class TestWebsiteSaleStockMultilang(HttpCase):
+    def test_website_sale_stock_multilang(self):
+        # Install French
+        website = self.env.ref('website.default_website')
+        lang_fr = self.env['res.lang']._activate_lang('fr_FR')
+        website.language_ids = self.env.ref('base.lang_en') + lang_fr
+        website.default_lang_id = self.env.ref('base.lang_en')
+        self.env['ir.module.module']._load_module_terms(
```

Comment:
Why do we need this?
We can't just create the translation for the product out of stock message?

Reply:
Indeed, now that the translation is hard-coded in the test, there is not need to load the module terms anymore.